api/src/main/java/org/openmrs/module/attachments/obs/Attachment.java

```diff
@@ -52,25 +52,9 @@ public Attachment(Obs obs) {
 		setVoided(obs.getVoided());
 		setVoidedBy(obs.getVoidedBy());
 		setVoidReason(obs.getVoidReason());
-
 		setDateTime(obs.getObsDatetime());
 		setComment(obs.getComment());
-
-		// Chomp the UUID off the end of the obs if any
```

Comment:
Had originally considered removing this, but at the end of the day I just moved this down into a utility method.